iced_layershell/src/multi_window.rs

```diff
@@ -362,6 +362,15 @@ where
                             };
                             window.set_size((width, height));
                         }
+                        LayershellCustomActions::ExclusiveZoneChange(zone_size) => {
+                            let Some(id) = id else {
```

Comment:
I think here should also log an error as the other place, because here the id should exist.

Reply:
Done. This code is written in Jan. So it didn't contain the log like other branches.